recipes/edrixs/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "edrixs" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fce7c1543d61df156696533dd7abcb7782e410bfb91afe7ae77109735146d160
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - make
+    - openblas
```

Comment:
Why is openblas in build? This should not be needed.

Reply:
Good catch! Probably it's a leftover from the previous attempts to make it work and to address missing libraries. I committed a fix, let's see if it works without it.

Reply:
Done, the build is successful. Could you please re-review it?